package.json

```diff
@@ -4,8 +4,8 @@
 	"author": "Intuita",
 	"packageManager": "pnpm@8.9.0",
 	"scripts": {
-		"build": "turbo run build --no-daemon",
-		"build:homedir": "turbo run build:homedir --no-daemon",
+		"build": "turbo run build:cjs && pnpm build --filter @codemod-registry/builder",
```

Comment:
can you switch the github runner configuration to ensure the `build:cjs` job is not run anymore directly?